packages/fxa-settings/src/components/Tooltip/index.tsx

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { ReactNode } from 'react';
+import classNames from 'classnames';
+
+export type TooltipType = 'default' | 'error';
+export type PositionType = 'top' | 'bottom';
+
+type TooltipProps = {
+  message: string;
+  type?: TooltipType;
+  anchorLeft?: boolean;
+  className?: string;
+  position?: PositionType;
+};
+
+export const Tooltip = ({
+  message,
+  className,
+  type = 'default',
+  anchorLeft = false,
+  position = 'top',
+}: TooltipProps) => {
+  const bgColor = type === 'error' ? 'red-600' : 'grey-500';
+  // caret position and the positioning rule for tooltip are actually the opposite
+  // of the passed in position;
+  const oppositePositionRule = {
+    top: 'bottom',
+    bottom: 'top',
+  }[position];
+
+  return (
+    <div
+      data-testid="tooltip"
+      title={message}
+      className={classNames(
+        `z-50 absolute py-2 px-6 text-center text-white
+         p-3 rounded text-xs left-0 font-Metropolis font-bold
```

Comment:
The class `font-Metropolis` does not exist, I think you are looking for `font-header` here.

Reply:
It looks like in the Figma doc both tooltips have a box shadow. Can we apply that here?